src/utils/update/fetch.beneficial.owners.private.data.ts

```diff
@@ -31,20 +31,19 @@ export const fetchBeneficialOwnersPrivateData = async (appData: ApplicationData,
 };
 
 const mapBeneficialOwnersPrivateData = (boPrivateData, appData: ApplicationData) => {
-  if (boPrivateData === undefined || boPrivateData.length === 0) {
-    return;
-  }
-  const mapUsualResidentialAddress = beneficialOwner => {
+  appData.update?.review_beneficial_owners_individual?.forEach(beneficialOwner => {
+    if (beneficialOwner.ch_reference) {
+      mapIndividualBOPrivateData(boPrivateData, beneficialOwner);
+    }
+  });
+  appData.update?.review_beneficial_owners_corporate?.forEach(beneficialOwner => {
     if (beneficialOwner.ch_reference) {
-      beneficialOwner.usual_residential_address = mapBoPrivateAddress(boPrivateData, beneficialOwner.ch_reference, false);
+      mapCorporateOrGovernmentBOPrivateData(boPrivateData, beneficialOwner);
     }
-  };
-  const mapPrincipalAddress = beneficialOwner => {
+  });
+  appData.update?.review_beneficial_owners_government_or_public_authority?.forEach(beneficialOwner => {
     if (beneficialOwner.ch_reference) {
-      beneficialOwner.principal_address = mapBoPrivateAddress(boPrivateData, beneficialOwner.ch_reference, true);
+      mapCorporateOrGovernmentBOPrivateData(boPrivateData, beneficialOwner);
     }
-  };
```

Comment:
we did these with closures due to duplicate code smell

Reply:
Yeah I changed it back....Draconian Sonarqube is what that is. For all it was.

Reply:
👍